streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -1568,10 +1548,8 @@ public void shouldNotShareHeadersBetweenPunctuateIterations() {
 
     @Test
     public void shouldWrapKafkaExceptionWithStreamsExceptionWhenProcess() {
-        EasyMock.expect(stateManager.changelogPartitions()).andReturn(Collections.emptySet()).anyTimes();
-        EasyMock.expect(stateManager.changelogOffsets()).andReturn(emptyMap()).anyTimes();
-        EasyMock.expect(recordCollector.offsets()).andReturn(emptyMap()).anyTimes();
-        EasyMock.replay(stateManager, recordCollector);
+        when(stateManager.changelogOffsets()).thenReturn(emptyMap());
+        when(recordCollector.offsets()).thenReturn(emptyMap());
```

Comment:
Do we even need those? What is the default return value for a map? I guess an empty map. Could you please verify?

Reply:
This is also absolutely correct, these are not needed. I have removed all stub-related references to emptyMap